.env.example

```diff
@@ -7,7 +7,14 @@ SESSION_SECRET="super-duper-s3cret"
 # https://letsencrypt.org/docs/expiration-emails/
 LETS_ENCRYPT_ACCOUNT_EMAIL="nx@senecacollege.ca"
 
-PORT=8080
-
 # One of: trace, debug, info, warn, error, fatal, silent
 LOG_LEVEL=debug
+
+PORT=8080
+
+# Nodemailer configuration
+NODEMAILER_USER="no-reply@senecacollege.ca"
+
+NODEMAILER_CLIENT_ID="secret-client-id"
```

Comment:
The client id and access token should be secrets we pull from Docker engine.  Take a look at how @dadolhay did it for Let's Encrypt in https://github.com/Seneca-CDOT/starchart/blob/main/app/lib/lets-encrypt.server.ts#L18.  You can do the same here.

This will read the values at runtime from a file that is mounted into the container by Docker.